Add Getting Started with PivotGrid (#2860)
DevExpress/devextreme-documentation: concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/08 Configure Fields and Areas.md

> To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures a single pivot grid field. Specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.

Reviewer: > To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures a single pivot grid field.

"A single" looks like a tautology. It reads as an extra emphasis on the "singularity" aspect:

> ... configures **just only one** pivot grid field.

Just "one" is more than enough here.

```suggestion
To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures one grid field. Specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
```

Author: Well, if it were "multiple fields" instead of "a single field", you wouldn't say that there's tautology, would you? The indefinite article is needed to satisfy grammar rules, nothing more. Anyway, I see that you understood the original perfectly well. Then, what's the point of editing it?

Reviewer: The point is that indefinite article already states "one" or "single". If you put "single", it places an extra emphasis on the singularity aspect. I only suggested "one" because it can replace the indefinite article while preserving your emphasis to some extent. Either way, I attracted your attention towards this fact, you can do with it whatever you deem necessary.

> You can distribute fields between four different areas: row, column, filter, and data. Row and column fields contain values displayed in row and column headers. Filter fields are invisible, but users can filter data against them. Data fields contain values for summary calculation. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. In the following code, we declare two row fields, two column fields, and two data fields:

Reviewer 2: ```suggestion
You can distribute fields between four different areas: rows, columns, filters, and data. Row and column fields contain values displayed in row and column headers. Filter fields are invisible, but users can filter data against them. Data fields contain values for summary calculation. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. In the following code, we declare two row fields, two column fields, and two data fields:
```

> You can distribute fields between four different areas: row, column, filter, and data. Row and column fields contain values displayed in row and column headers. Filter fields has no visual representation, but they can be used to filter data. Data fields contain values for summary calculation. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. In the following code, we declare two row fields, two column fields, and two data fields:

Reviewer: > Filter fields **has** no visual **representation**, but they can be used to filter data.

1. Not "has" but "have".
2. I've grown to fear the root "present" in our texts, moreover in the "represent" form.

 Can we rewrite this sentence as follows?
 
 > Filter fields are invisible but you can filter data against them.

Author: ```suggestion
You can distribute fields between four different areas: row, column, filter, and data. Row and column fields contain values displayed in row and column headers. Filter fields are invisible, but users can filter data against them. Data fields contain values for summary calculation. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. In the following code, we declare two row fields, two column fields, and two data fields:
```
